docs/Gemfile

```diff
@@ -3,3 +3,5 @@ source 'https://rubygems.org'
 gem 'github-pages'
 
 gem "webrick", "~> 1.7"
+
+gem "jekyll", "~> 3.9"
```

Comment:
Do we have to explicitly specify `jekyll` here? The docs for `github-pages` state the following:

> Note: You are not required to install Jekyll separately. Once the github-pages gem is installed, you can build your site using jekyll build, or preview your site using jekyll serve. For more information about installing Jekyll locally, please see [the GitHub Help docs on the matter](https://help.github.com/articles/using-jekyll-with-pages#installing-jekyll).

Reply:
@jonkoops requested me to add it https://github.com/Leaflet/Leaflet/pull/8504#discussion_r985103621

Reply:
I'll try adding a commit without it . Let's see if it passes

Reply:
Oh nice, I was not aware it was a transitive dependency.